BTCPayServer/Payments/Changelly/Changelly.cs

```
@@ -0,0 +1,135 @@
+                else
+                {
+                    var appendedResult = _showFiat
+                        ? result.Result.Result.Concat(new[]
+                        {
+                            new CurrencyFull()
+                            {
+                                Enable = true,
+                                Name = "EUR",
+                                FullName = "Euro",
+                                PayInConfirmations = 0,
+                                ImageLink = "https://changelly.com/api/coins/eur.png" 
+                            },
+                            new CurrencyFull()
+                            {
+                                Enable = true,
+                                Name = "USD",
+                                FullName = "US Dollar",
+                                PayInConfirmations = 0,
+                                ImageLink = "https://changelly.com/api/coins/usd.png" 
+                            }
+                        })
+                        : result.Result.Result;
+                    return (appendedResult, true, "");
+                }
+            }
+            catch (Exception Ex)
+            {
+                return (null, false, Ex.Message);
+            }
+        }
+
+        public virtual async Task<(decimal Amount, bool Success, string Error)> GetExchangeAmount(string fromCurrency,
+            string toCurrency,
+            decimal amount)
+        {
+            try
+            {
+                var message =
+                    "{\"id\": \"test\",\"jsonrpc\": \"2.0\",\"method\": \"getExchangeAmount\",\"params\":{\"from\": \"" +
+                    fromCurrency + "\",\"to\": \"" + toCurrency + "\",\"amount\": \"" + amount + "\"}}";
+
+                var result = await PostToApi<string>(message);
+                return !result.Success ? (0, false, result.Error) : (Convert.ToDecimal(result.Result.Result), true, "");
```

Instead of returning error Go-style I think it is better to throw a proper `ChangellyException`